### ISSUE TITLE
Multiple onFlutterError calls

### ISSUE THREAD

**Issue by StarProxima** [MadBrains/App-Runner-Flutter#14]
On hot reload, onFlutterError is written to FlutterError.onError along with oldCallback. Because of this, every hot reload, another onFlutterError is added to FlutterError.onError and its call occurs several times with one error.

Reproduced in the project example

When commenting on this line, everything is corrected, but oldCallback is not taken into account

```dart
class _App extends StatelessWidget {
  const _App({
    Key? key,
    required this.widgetConfig,
  }) : super(key: key);

  final WidgetConfiguration widgetConfig;

  @override
  Widget build(BuildContext context) {
    return ReloadableWidget(
      builder: (BuildContext context) {
        _flutterErrorSetup();
        ErrorWidget.builder = (FlutterErrorDetails errorDetails) => ErrorHandlerWidget(
              errorDetails: errorDetails,
              releaseErrorBuilder: widgetConfig.releaseErrorBuilder,
              errorBuilder: widgetConfig.errorBuilder,
            );

        return widgetConfig.child;
      },
    );
  }

  void _flutterErrorSetup() {
    final FlutterExceptionHandler? oldCallback = FlutterError.onError;

    FlutterError.onError = (FlutterErrorDetails errorDetails) {
      // oldCallback?.call(errorDetails);   <-- This line
      widgetConfig.onFlutterError(errorDetails);
    };
  }
}

```

```
$ flutter --version
Flutter 3.10.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 796c8ef792 (4 weeks ago) • 2023-06-13 15:51:02 -0700
Engine • revision 45f6e00911
Tools • Dart 3.0.5 • DevTools 2.23.1
```

**Comment by StarProxima** [comment]
@KolasikOmetov Could you watch this?

**Comment by KolasikOmetov** [comment]
This issue was fixed in 2.2.2 version.
